SplashScreen/app/src/main/AndroidManifest.xml

```diff
@@ -24,12 +24,13 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.SplashScreen">
+        android:theme="@style/Theme.App">
 
         <activity
             android:name=".DefaultActivity"
             android:exported="true"
-            android:label="@string/default_splash_screen">
+            android:label="@string/default_splash_screen"
+            android:theme="@style/Theme.App.SplashScreen">
```

Comment:
Can you add a comment to explain that if the application has multiple entry points, it's better to set the SplashScreen theme on the <application> tag?

Reply:
Done.